### PR TITLE
feat: add skopos metadata and icon

### PR DESCRIPTION
**Pull request by J4yJ4y-R6**
- [x] Add Skopos icon
- [x] Add Skopos information

If you want me to change something, pls let me know.

### CODE REVIEW

File: operators/skopos/index.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  role: "Defender",
+  unit: "WOLFGUARD",
+  ratings: {
+    health: 2,
+    speed: 2,
+    difficulty: 2,
+  },
+  meta: {
+    gender: "f",
+    country: "Greek",
```
[the remark this answers is not in the document]
Comment (AlmostSuspense) [reply operators/skopos/index.ts:14]: i believe locale codes are used instead of country names

**Review by AlmostSuspense — CHANGES_REQUESTED** [review]

File: operators/skopos/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (J4yJ4y-R6) [reply operators/skopos/index.ts]: Thanks! I forgot about that :)

**Comment by SwiftCODA** [comment]
Bump @marcopixel ?

**Comment by marcopixel** [comment]
@SwiftCODA @J4yJ4y-R6 

Sorry, i'm quite busy right now so i haven't had the time to check the changes yet.
I'll try to get this checked out on the weekend, so please hang in for now 🙏

**Pull request merged**